src/support/thread_group.c

```diff
@@ -181,8 +182,9 @@ __thread_group_resize(WT_SESSION_IMPL *session, WT_THREAD_GROUP *group, uint32_t
         WT_ERR(__wt_calloc_one(session, &thread));
         /* Threads get their own session. */
         session_flags = LF_ISSET(WT_THREAD_CAN_WAIT) ? WT_SESSION_CAN_WAIT : 0;
+        WT_ERR(__wt_snprintf(session_name, sizeof(session_name), "%s-%" PRIu32, group->name, i));
         WT_ERR(
-          __wt_open_internal_session(conn, group->name, false, session_flags, 0, &thread->session));
+          __wt_open_internal_session(conn, session_name, false, session_flags, 0, &thread->session));
```

Comment:
This seems to have an issue, since session_name is local array, whereas WT_SESSION_IMPL.name is `char*`. Can I make it array or malloc when session opened ?